enterprise/ha/src/main/java/org/neo4j/kernel/ha/lock/forseti/ForsetiLockManager.java

```diff
@@ -161,7 +164,7 @@ public void accept( Visitor out )
                 for ( Map.Entry<Long, Lock> entry : lockMaps[i].entrySet() )
                 {
                     Lock lock = entry.getValue();
-                    out.visit( type, entry.getKey(), lock.describeWaitList(), 0 );
+                    out.visit( type, entry.getKey(), lock.describeWaitList(), 0, System.identityHashCode( lock ) );
```

Comment:
Need to recognise in some way lock identity and since `SharedLock` already used `System.identityHashCode` in some sort for identity, visitor use the same way